app/src/main/java/org/oppia/android/app/help/HelpActivityPresenter.kt

```diff
@@ -110,17 +178,108 @@ class HelpActivityPresenter @Inject constructor(private val activity: AppCompatA
+      LICENSE_LIST_FRAGMENT_TAG -> handleLoadLicenseListFragment(dependencyIndex)
+      LICENSE_TEXT_FRAGMENT_TAG -> handleLoadLicenseTextViewerFragment(
+        dependencyIndex,
+        licenseIndex
+      )
+    }
+  }
+
+  private fun selectFAQListFragment() {
+    setMultipaneContainerTitle(activity.getString(R.string.faq_activity_title))
+    setMultipaneBackButtonVisibility(View.GONE)
+    selectedFragmentTag = FAQ_LIST_FRAGMENT_TAG
+    selectedHelpOptionTitle = getMultipaneContainerTitle()
+  }
+
+  private fun selectThirdPartyDependencyListFragment() {
+    setMultipaneContainerTitle(
+      activity.getString(R.string.third_party_dependency_list_activity_title)
+    )
+    setMultipaneBackButtonVisibility(View.GONE)
+    selectedFragmentTag = THIRD_PARTY_DEPENDENCY_LIST_FRAGMENT_TAG
+    selectedHelpOptionTitle = getMultipaneContainerTitle()
+  }
+
+  private fun selectLicenseListFragment(dependencyIndex: Int) {
+    setMultipaneContainerTitle(activity.getString(R.string.license_list_activity_title))
+    setMultipaneBackButtonVisibility(View.VISIBLE)
+    setHelpBackButtonContentDescription(LICENSE_LIST_FRAGMENT_TAG)
+    selectedFragmentTag = LICENSE_LIST_FRAGMENT_TAG
+    selectedDependencyIndex = dependencyIndex
+    selectedHelpOptionTitle = getMultipaneContainerTitle()
+  }
+
+  private fun selectLicenseTextViewerFragment(dependencyIndex: Int, licenseIndex: Int) {
+    setMultipaneContainerTitle(retrieveLicenseName(dependencyIndex, licenseIndex))
+    setMultipaneBackButtonVisibility(View.VISIBLE)
+    setHelpBackButtonContentDescription(LICENSE_TEXT_FRAGMENT_TAG)
+    selectedFragmentTag = LICENSE_TEXT_FRAGMENT_TAG
+    selectedDependencyIndex = dependencyIndex
+    selectedLicenseIndex = licenseIndex
+    selectedHelpOptionTitle = getMultipaneContainerTitle()
+  }
+
+  private fun retrieveLicenseName(dependencyIndex: Int, licenseIndex: Int): String {
+    val thirdPartyDependencyLicenseNamesArray = activity.resources.obtainTypedArray(
+      R.array.third_party_dependency_license_names_array
+    )
+    val licenseNamesArrayId = thirdPartyDependencyLicenseNamesArray.getResourceId(
+      dependencyIndex,
+      /* defValue= */ 0
+    )
+    val licenseNamesArray = activity.resources.getStringArray(licenseNamesArrayId)
+    thirdPartyDependencyLicenseNamesArray.recycle()
+    return licenseNamesArray[licenseIndex]
+  }
+
+  private fun getMultipaneContainerTitle(): String {
+    return activity.findViewById<TextView>(
+      R.id.help_multipane_options_title_textview
+    ).text.toString()
+  }
+
+  private fun setHelpBackButtonContentDescription(fragmentTag: String) {
+    when (fragmentTag) {
+      LICENSE_LIST_FRAGMENT_TAG -> {
+        activity.findViewById<ImageButton>(R.id.help_multipane_options_back_button)
+          .contentDescription = String.format(
+          activity.getString(
+            R.string.help_activity_back_arrow_description,
+            "third-party dependencies list"
```

Comment:
What is the use of these format arguments? `"third-party dependencies list"`

Reply:
As per your [earlier comment](https://github.com/oppia/oppia-android/pull/3671#issuecomment-899274381), I am setting the content description dynamically.

Reply:
Please don't set it in hard-code form otherwise we won't be able to translate this to other languages. These strings should be defined in `strings.xml` file and then should be used here.

Reply:
Done.